### PR TITLE
Improve branding

### PR DESCRIPTION
**Pull request by mario**
This improves branding by:

- removing useless options
- introducing new ones
- making some work :D

### CODE REVIEW

File: src/main/res/values/styles.xml [hunk 1]
```diff
@@ -129,10 +129,10 @@
 
 	<!-- ACTION BAR STYLES -->
 	<style name="Theme.ownCloud.Overlay.ActionBar" parent="@style/Widget.AppCompat.ActionBar">
-		<item name="android:background">@color/owncloud_blue_transparent</item>
+		<item name="android:background">@color/color_transparent</item>
 		<item name="android:windowActionBarOverlay">true</item>
 		<!-- Support library compatibility -->
-		<item name="background">@color/owncloud_blue_transparent</item>
+		<item name="background">@color/transparent</item>
```
Comment (AndyScherzinger) [inline src/main/res/values/styles.xml]: Shouldn't this also be color_transparent?

File: src/modified/res/values/setup.xml [hunk 2]
```diff
@@ -35,18 +35,24 @@
...
+    <color name="primary_dark">#006AA3</color>
+    <color name="color_accent">@color/infolevel_critical</color>
     <color name="login_text_color">@color/white</color>
     <color name="login_text_hint_color">#7fC0E3</color>
     <color name="login_background_color">#FFFFFF</color>
     <color name="login_logo_background_color">#FFFFFF</color>
     <color name="background_color">#FFFFFF</color>
-    <color name="actionbar_start_color">#0082c9</color>
-    <color name="actionbar_end_color">#0082c9</color>
+    <color name="navigation_bar_start_color">@color/infolevel_critical</color>
```
Comment (AndyScherzinger) [inline src/modified/res/values/setup.xml]: Why are the colors for the action bar named info level critical?

File: src/modified/res/values/setup.xml [hunk 3]
```diff
@@ -35,18 +35,24 @@
     <bool name = "share_with_users_feature">true</bool>
 
     <!-- Colors -->
+    <color name="primary">@color/owncloud_blue</color>
+    <color name="primary_dark">#006AA3</color>
+    <color name="color_accent">@color/infolevel_critical</color>
```
Comment (AndyScherzinger) [inline src/modified/res/values/setup.xml]: Same here, the info level should refer to the right color, which is accent

File: src/main/res/values/setup.xml [hunk 4]
```diff
@@ -39,18 +39,24 @@
     <bool name="show_whats_new">true</bool>
     
     <!-- Colors -->
+    <color name="primary">@color/owncloud_blue</color>
```
Comment (AndyScherzinger) [inline src/main/res/values/setup.xml]: Can we rename it to loose the owncloud prefix

**Comment by AndyScherzinger** [comment]
:+1: Looks good to me, just some minor issues :)

File: src/main/res/values/styles.xml (hunk 1 above)
Comment (mario) [inline src/main/res/values/styles.xml]: Possibly :p

File: src/modified/res/values/setup.xml (hunk 3 above)
Comment (mario) [inline src/modified/res/values/setup.xml]: Notice that it's in modified. I just made it different to the original one :) Is that an issue?

File: src/modified/res/values/setup.xml (hunk 2 above)
Comment (mario) [inline src/modified/res/values/setup.xml]: So they're different than the original one.

File: src/main/res/values/setup.xml (hunk 4 above)
Comment (mario) [inline src/main/res/values/setup.xml]: sure ...

**Comment by mario** [comment]
@AndyScherzinger updated all to your desires.

File: src/modified/res/values/setup.xml (hunk 3 above)
Comment (AndyScherzinger) [inline src/modified/res/values/setup.xml]: Ah, missed that... Ups... ;)

**Comment by AndyScherzinger** [comment]
:+1: sweet work!

**Comment by tobiasKaminsky** [comment]
👍

**Pull request merged**